substrate/primitives/consensus/beefy/src/lib.rs

```diff
@@ -257,6 +260,7 @@ pub enum ConsensusLog<AuthorityId: Codec> {
 ///
 /// A vote message is a direct vote created by a BEEFY node on every voting round
 /// and is gossiped to its peers.
+/// TODO: Remove `Signature` generic type, instead get it from `Id::Signature`.
```

Comment:
```suggestion
// TODO: Remove `Signature` generic type, instead get it from `Id::Signature`.
```
Hide TODOs from docs